packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.html

```diff
@@ -0,0 +1,216 @@
+            <span
+                ><strong
+                    data-ng-bind="$ctrl.getSwitchBtnStatusText($ctrl.model.http.enabled)"
+                ></strong
+            ></span>
+        </div>
+
+        <!--DEV-MODE-->
+        <div class="mb-4">
+            <p class="oui-heading_5 mb-1"><strong>Dev-mode</strong></p>
+            <div
+                class="oui-paragraph"
+                data-translate="hosting_cdn_shared_option_always_online_info"
+            ></div>
+            <oui-switch
+                disabled="!$ctrl.model.devmode"
+                data-model="$ctrl.model.devmode.enabled"
+                data-on-change="$ctrl.model.devmode.changed = true"
+            >
+            </oui-switch>
+            <span
+                ><strong
+                    data-ng-bind="$ctrl.getSwitchBtnStatusText($ctrl.model.devmode.enabled)"
+                ></strong
+            ></span>
+        </div>
+
+        <!--BROTLI-->
+        <div class="mb-4">
+            <p class="oui-heading_5 mb-1"><strong>Brotli</strong></p>
+            <div
+                class="oui-paragraph"
+                data-translate="hosting_cdn_shared_option_always_online_info"
+            ></div>
+            <oui-switch
+                disabled="!$ctrl.model.brotli"
+                data-model="$ctrl.model.brotli.enabled"
+                data-on-change="$ctrl.model.brotli.changed = true"
+            >
+            </oui-switch>
+            <span
+                ><strong
+                    data-ng-bind="$ctrl.getSwitchBtnStatusText($ctrl.model.brotli.enabled)"
+                ></strong
+            ></span>
+        </div>
+
+        <!--CACHE RULE-->
+        <div class="mb-4">
+            <p class="oui-heading_5 mb-1"><strong>Cache rule</strong></p>
+            <div
+                class="oui-paragraph"
+                data-translate="hosting_cdn_shared_option_always_online_info"
+            ></div>
+            <oui-button
+                class="mb-3"
+                data-variant="secondary"
+                icon-left="oui-icon-plus"
+                data-on-click="$ctrl.openCreatCacheRuleModal()"
+            >
+                <span
+                    data-translate="hosting_cdn_shared_option_cache_rule_btn_add_rule"
+                ></span>
+            </oui-button>
+            <div>
+                <table class="oui-table">
+                    <thead class="oui-table__headers">
+                        <tr>
+                            <th
+                                class="oui-table__header oui-table__header_sortable"
+                                tabindex="0"
+                                data-translate="hosting_cdn_shared_option_cache_rule_table_header_order_by"
+                            >
+                                <span
+                                    class="oui-icon oui-icon-sort-inactive"
+                                    aria-hidden="true"
+                                ></span>
```

Comment:
There should be a description for screen readers

Reply:
we dont need it.
removed.